ideahacks/db/data/team.js

```diff
@@ -4,209 +4,215 @@
 
 module.exports = [
 	{
-		departmentName: "Directors",
-		teamPath: "/img/team/directors.png",
-		teamPathRotated: "/img/team/directors-rotated.png",
+		departmentName: "Team Captains",
+		isFourWide: false,
 		teammates: [
 			{
-				name: "Albert Han",
-				picturePath: "/img/team/albert.jpg"
+				name: "Solaine Zhao",
+				picturePath: "/img/team/solaine.jpg"
 			},
 			{
 				name: "Tao Lu",
 				picturePath: "/img/team/tao.jpg"
 			},
 			{
-				name: "Jay Park",
-				picturePath: "/img/team/jay.jpg"
+				name: "Pranav Srinivasan",
+				picturePath: "/img/team/pranav.jpg"
 			}
 		]
 	},
 	{
-		departmentName: "Design",
-		teamPath: "/img/team/design.png",
-		teamPathRotated: "/img/team/design-rotated.png",
-		teamLead: "Solaine Zhao",
+		departmentName: "Website",
+		isFourWide: false,
+		teamLead: "Robert Peralta",
 		teammates: [
 			{
-				name: "Solaine Zhao",
-				picturePath: "/img/team/solaine.jpg"
+				name: "Robert Peralta",
+				picturePath: "/img/team/robert.jpeg"
 			},
 			{
-				name: "Olivia Loh",
-				picturePath: "/img/team/olivia.jpg"
+				name: "Ava Asmani",
+				picturePath: "/img/team/justin.jpg"
 			},
 			{
-				name: "Anna Anderson",
-				picturePath: "/img/team/anna.jpeg"
+				name: "Grace Ma",
+				picturePath: "/img/team/james.jpg"
 			},
 			{
-				name: "Andrew Fantino",
-				picturePath: "/img/team/andrew.jpg"
+				name: "Yolanda Chen",
+				picturePath: "/img/team/rex.jpg"
 			},
 			{
-				name: "Ryan Tong",
-				picturePath: "/img/team/ryan.jpg"
+				name: "Raj Piskala",
+				picturePath: "/img/team/ray.jpg"
+			},
+			{
+				name: "Timothy Jacques",
+				picturePath: "/img/team/sahil.jpg"
 			}
 		]
 	},
 	{
-		departmentName: "Corporate",
-		teamPath: "/img/team/corporate.png",
-		teamPathRotated: "/img/team/corporate-rotated.png",
-		teamLead: "Pranav Srinivasan and Daniel Possemato",
+		departmentName: "Food",
+		isFourWide: false,
+		teamLead: "Jack Lin",
 		teammates: [
 			{
-				name: "Pranav Srinivasan",
-				picturePath: "/img/team/pranav.jpg"
-			},
-			{
-				name: "Daniel Possemato",
-				picturePath: "/img/team/daniel.png"
+				name: "Jack Lin",
```

Comment:
A lot of the images don't match the names. Is it possible for you to create a green square as a png, e.g. filler.png, and then use that as filler for people whose pictures we don't have yet? This way it's less likely the wrong picture will pass into production.

Reply:
Right now, none of the pictures are correct. I think there are rumors of an eventual photoshoot, so I'd assume that *all* of the photos are going to be replaced. I'll have to go through all of them again when the time comes, so I don't think this is an issue for now.

Reply:
I agree with Tim here; using last years' pictures as placeholders is traditionally done on this website (although Raj makes a good point that it's bad practice). We'll just leave them as is, as we will be doing a full pass over everything at a later date